webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/components/NodeGroupForm.scala

```diff
@@ -222,10 +222,15 @@ class NodeGroupForm(
       & "group-save" #> { if (CurrentUser.checkRights(AuthorizationType.Group.Edit))
                     <span class="save-tooltip-container">
                       {
-                        SHtml.ajaxButton(<span>Save <i class="fa fa-download"></i></span>, onSubmit _)  %  ("id" -> saveButtonId) % ("class" -> " btn btn-success btn-icon")
+                        SHtml.ajaxOnSubmit(onSubmit _)(
+                          <button class="btn btn-success btn-icon ui-button ui-corner-all ui-widget" id="groupSaveButtonId">
```

Comment:
I think you should try to give the id attribute from the saveButtonId val

try something like id="${saveButtonId}"

If saveButtonId was only used here, you may delete it i would be quite happy too

Reply:
It's used in other places